CI/travis.install.sh

```diff
@@ -1,8 +1,4 @@
 #!/bin/bash
-if [ -z "${TRAVIS_OS_NAME}" ] || [ "${TRAVIS_OS_NAME}" = "linux" ]; then
-  echo Install on linux.
```

Comment:
As above, keep conditional and change to:

    echo Install on linux - currently no travis.linux.install.sh present or expected